backend/src/game/constants.py

```diff
@@ -1,2 +1,13 @@
+import os
+
+# directories
+CARDS_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+# ints
 DEFAULT_HEALTH = 10
 ZERO = 0
+CARD_STORE_SIZE = 3
+JSON_CARD_COUNT = 66
```

Comment:
Do we need this in prod code? 
It looks like this is only used in the tests, in which case this should exist in a constants file that lives with or even in the test file.

Reply:
Oh duh. I'll fix that this week